journal/journal.go

```diff
@@ -53,11 +53,18 @@ const (
 var conn net.Conn
 
 func init() {
+	Reconnect()
+}
+
+// Estabish a new connection to the journald socket and returns a boolean
+// indicating if it has succeeded or failed.
+func Reconnect() bool {
 	var err error
 	conn, err = net.Dial("unixgram", "/run/systemd/journal/socket")
 	if err != nil {
 		conn = nil
 	}
+	return conn != nil
```

Comment:
Right, I was undecided while working on it, and didn't modify this part of code as it was unused.
I'll have another look at it to update the patch.